### PR TITLE
Display room title

### PR DESCRIPTION
**Pull request by enokawa**
Closes: N/A

## Description

Display the room title on the room page.

## Details of this change

Call the Find API to API Gateway from Client and some modified design.

## Information for Reviewer

The revised design is as follows:

room

### CODE REVIEW

**Comment by enokawa** [comment]
@nari-z Please review!!

File: frontend/src/pages/ChatPage.vue [hunk 1]
```diff
@@ -1,52 +1,52 @@
 <template>
   <v-container>
-    <h1>Chat Page, Room Title</h1>
+    <h1>{{ roomInfo.title }}</h1>
     <v-row>
       <v-col cols="12">
-        <li v-for="message in messages" :key="message.title">
+        <li v-for="message in messages" :key="message.title" class="pb-1">
```
Comment (nari-z) [inline frontend/src/pages/ChatPage.vue]: [MUST]
```suggestion
        <li v-for="message in messages" :key="message.uuid" class="pb-1">
```

Comment (nari-z) [reply frontend/src/pages/ChatPage.vue]: [Q]
whats `pb-1`?

File: frontend/src/pages/ChatPage.vue [hunk 2]
```diff
@@ -1,52 +1,52 @@
 <template>
   <v-container>
-    <h1>Chat Page, Room Title</h1>
+    <h1>{{ roomInfo.title }}</h1>
```
Comment (nari-z) [inline frontend/src/pages/ChatPage.vue:3]: 👍

File: frontend/src/pages/ChatPage.vue [hunk 3]
```diff
@@ -1,52 +1,52 @@
...
       <v-col cols="12">
-        <li v-for="message in messages" :key="message.title">
+        <li v-for="message in messages" :key="message.title" class="pb-1">
           <v-card
             outlined
           >
-            <v-list-item three-line style="text-align: left;">
+            <v-list-item single-line style="text-align: left;">
               <v-list-item-content>
-                <v-list-item-title >{{ message.title }}</v-list-item-title>
```
Comment (nari-z) [inline frontend/src/pages/ChatPage.vue:12]: [IMO]
Shouldn't we display something on title?(ex. date, message index)

File: frontend/src/pages/ChatPage.vue [hunk 4]
```diff
@@ -1,52 +1,52 @@
...
   export default {
     name: 'ChatPage',
 
     data: () => ({
       messages: [],
       postMessage: "",
       socket: null,
-      state: "idle"
+      state: "idle",
+      roomInfo: []
```
Comment (nari-z) [inline frontend/src/pages/ChatPage.vue]:
```suggestion
      roomInfo: {}
```

File: frontend/src/App.vue [hunk 5]
```diff
@@ -26,6 +26,19 @@
...
+    <v-footer
+      app
+      color="primary"
+      dark
+    >
+      <v-col
+        class="text-center"
+        cols="12"
+      >
+        {{ new Date().getFullYear() }} - <strong>tmpchat</strong>
```
Comment (nari-z) [inline frontend/src/App.vue]: [IMO]
How about defining date in data?

File: frontend/src/pages/ChatPage.vue [hunk 6]
```diff
@@ -96,13 +96,24 @@
       receiveMessage(event) {
         console.log('Message from server ', event.data);
         this.messages.push({
-          title: "Example",
           value: event.data
         });
+      },
+      getRoom(roomId) {
```
Comment (nari-z) [inline frontend/src/pages/ChatPage.vue]: [IMO]
Think this function is not `getRoom` because no return value.

**Review by nari-z — CHANGES_REQUESTED** [review]
Request Change

File: frontend/src/pages/ChatPage.vue (hunk 1 above)
Comment (enokawa) [reply frontend/src/pages/ChatPage.vue]: The `pb-1` is **p**adding **b**ottom - 4px . See https://vuetifyjs.com/ja/styles/spacing/ .

File: frontend/src/pages/ChatPage.vue (hunk 3 above)
Comment (enokawa) [reply frontend/src/pages/ChatPage.vue:12]: I agree. It will implement in another PR.

File: frontend/src/pages/ChatPage.vue (hunk 4 above)
Comment (enokawa) [reply frontend/src/pages/ChatPage.vue]: That's right. Fixed in b6d0c98f .

File: frontend/src/App.vue (hunk 5 above)
Comment (enokawa) [reply frontend/src/App.vue]: Fixed in ddd534c4 .

File: frontend/src/pages/ChatPage.vue (hunk 6 above)
Comment (enokawa) [reply frontend/src/pages/ChatPage.vue]: I couldn't understand it. Tell me more about it. I think this function does indeed get.  
So is there a better way to name it?

File: frontend/src/pages/ChatPage.vue (hunk 1 above)
Comment (nari-z) [reply frontend/src/pages/ChatPage.vue]: thanx! understand.

Comment (nari-z) [reply frontend/src/pages/ChatPage.vue]: did you fix this?
https://github.com/tmpchat/tmpchat/pull/39#discussion_r483149456

Comment (enokawa) [reply frontend/src/pages/ChatPage.vue]: Oops. Fixed in b37d051a .

File: frontend/src/pages/ChatPage.vue (hunk 6 above)
Comment (enokawa) [reply frontend/src/pages/ChatPage.vue]: Fixed in 35bc1ecd .

**Review by nari-z — APPROVED** [review]
LGTM

**Pull request merged**